src/views/Card/Card.js

```diff
@@ -2,13 +2,29 @@ import cx from 'classnames'
 import React, { PropTypes } from 'react'
 
 import META from '../../utils/Meta'
+import * as sui from '../../utils/semanticUtils'
+import { useKeyOnly } from '../../utils/propUtils'
 
 function Card(props) {
-  const { children, className, ...rest } = props
-  const classes = cx(
+  const {
+    centered,
+    children,
     className,
+    color,
+    fluid,
+    raised,
+    ...rest,
```

Comment:
We have another `propUtil` to get the `rest`:
```jsx
import { getUnhandledProps } from '../../utils/propUtils'

const rest = getUnhandledProps(Card, props)
```

This is needed since we might not always destructure _all_ the props when using `...rest`.  Especially when there are multiple render methods.  The util creates an object of all the props that your component does not handle (i.e. the rest of the user's props).